static/css/section/_things.scss

```diff
@@ -0,0 +1,314 @@
+@charset 'UTF-8';
```

Comment:
Are you intentionally adding this file? Was this a bad merge? I can't find mention if it being included anywhere, or of the `internet-of-things` class being used.